lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -111,14 +111,24 @@ def self.process_network_mask(network)
     # Useful for network, redistribute, etc.
     #   should: an array of expected cmds (manifest/recipe)
     #  current: an array of existing cmds on the device
```

Comment:
These comments should be at the top of the delta_add_remove method instead of here. Or just move the depth method to follow delta_add_remove.